model-client/src/jvmMain/kotlin/org/modelix/model/client/RestWebModelClient.kt

```diff
@@ -371,6 +371,10 @@ class RestWebModelClient @JvmOverloads constructor(
         return runBlocking { getA(key) }
     }
 
+    override fun getIfCached(key: String): String? {
+        return null
```

Comment:
Some comment that caching is not implemented here? Or why is `null` valid?

Reply:
Added comment.